src/Adapter/File/Uploader/AttachmentFileUploader.php

```diff
@@ -106,23 +111,13 @@ private function deleteOldFile(int $attachmentId, $throwExceptionOnFailure = tru
     }
 
     /**
-     * @param string $filePath
-     * @param string $uniqid
-     * @param int $fileSize
-     *
      * @throws AttachmentConstraintException
      * @throws AttachmentUploadFailedException
      */
     private function uploadFile(string $filePath, string $uniqid, int $fileSize): void
     {
         if ($fileSize > ($this->configuration->get('PS_ATTACHMENT_MAXIMUM_SIZE') * 1024 * 1024)) {
-            throw new AttachmentConstraintException(sprintf(
-                'Max file size allowed is "%s" bytes. Uploaded file size is "%s".',
-                    (string) ($this->configuration->get('PS_ATTACHMENT_MAXIMUM_SIZE') * 1024),
-                    number_format(($fileSize / 1024), 2, '.', '')
-                ),
-                AttachmentConstraintException::INVALID_FILE_SIZE
-            );
+            throw new AttachmentConstraintException(sprintf('Max file size allowed is "%s" bytes. Uploaded file size is "%s".', (string) ($this->configuration->get('PS_ATTACHMENT_MAXIMUM_SIZE') * 1024), number_format(($fileSize / 1024), 2, '.', '')), AttachmentConstraintException::INVALID_FILE_SIZE);
```

Comment:
This should be broken up in several lines.

Reply:
Fixed